user_guide_src/source/installation/upgrade_4211.rst

```diff
@@ -0,0 +1,31 @@
+##############################
+Upgrading from 4.2.10 to 4.2.11
+##############################
```

Comment:
I'm not sure if this needs updating of the over and under lines?
```suggestion
###############################
Upgrading from 4.2.10 to 4.2.11
###############################
```

Reply:
My view from the browser tells me it needs updating. Not sure from local view.

Reply:
It needs.
> user_guide_src/source/installation/upgrade_4211.rst:1: WARNING: Title overline too short.

Reply:
Probably?